lib/LaTeXML/Post/MathML.pm

```diff
@@ -1071,6 +1024,210 @@ sub pmml_text_aux {
   else {
     return (); } }
 
+#%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+# Adjust the spacing by comparing TeX & author specified spacing to MathML's defaults
+sub adjust_spacing {
+  my ($self, $mml) = @_;
+  space_walk($self, $mml);
```

Comment:
A name for the ages. 🛰️ 👨🏻‍🚀 🌎

Reply:
An interesting observation -- the `adjust_spacing` routine does exactly a single call to `space_walk` and nothing else. So they're interchangeable. And possibly redundant? If one wanted to always use one of the names or the other, one could?

Reply:
It was initially proactive, in case weird arguments, `local` etc were needed. Then I kept it because the inner function name is too much fun, but too unclear at the "top" level.